phys2bids/tests/test_physio_obj.py

```diff
@@ -0,0 +1,82 @@
+"""
+Tests physio_obj.py
+"""
+
+import numpy as np
+from phys2bids import physio_obj as po
+
+
+# Tests is_valid
+def test_is_valid():
+    test_list = [0, 1, 1, 2, 3, 5, 8, 13]
+    valid_output = po.is_valid(test_list, list, int)
+    assert valid_output == test_list
+
+# Tests has_size
+def test_has_size():
+    test_list = [0, 1, 1, 2, 3, 5, 8, 13]
+    test_length = 10
+    size_output = po.has_size(test_list, test_length, 0)
+    assert len(size_output) == test_length
```

Comment:
Can we maybe add another check here to make sure that the data aren't padded if they're already the correct size? Also a few checks to make sure that we're padding the data with the correct fill value would be awesome!